stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanActivity.kt

```diff
@@ -256,15 +256,10 @@ internal class CardScanActivity : ScanActivity(), SimpleScanStateful<CardScanSta
         viewBinding.swapCameraButton.setVisible(supported)
     }
 
-    /**
-     * Prepare to start the camera. Once the camera is ready, [onCameraReady] must be called.
-     */
-    override fun prepareCamera(onCameraReady: () -> Unit) {
-        viewBinding.previewFrame.post {
-            viewBinding.viewFinderBackground
-                .setViewFinderRect(viewBinding.viewFinderWindow.asRect())
-            onCameraReady()
-        }
+    override fun onCameraReady() {
+        viewBinding.viewFinderBackground
```

Comment:
Nice! If this is driven by the permission, we don't have to wait for the `previewFrame` to become available.